modules/nf-core/pilon/main.nf

```diff
@@ -39,5 +39,20 @@ process PILON {
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         pilon: \$(echo \$(pilon --version) | sed 's/^.*version //; s/ .*\$//' )
+    END_VERSIONS
     """
+
+    stub:
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def valid_mode = ["frags", "jumps", "unpaired", "bam"]
+    if ( !valid_mode.contains(pilon_mode) )  { error "Unrecognised mode to run Pilon. Options: ${valid_mode.join(', ')}" }
+    """
+    touch ${prefix}.fasta
```

Comment:
I would suggest emitting all files even if optional, this would allow easier chaining of modules in a subwf/pipeline stub run, if a downstream module wants one of the optional outputs